packages/generator/templates/app/app/pages/_app.tsx

```diff
@@ -9,19 +9,24 @@ import {
 } from "blitz"
 import { ErrorBoundary } from "react-error-boundary"
 import LoginForm from "app/auth/components/LoginForm"
+import { Suspense } from "react"
 
 export default function App({ Component, pageProps }: AppProps) {
   const getLayout = Component.getLayout || ((page) => page)
   const router = useRouter()
 
   return (
-    <ErrorBoundary
-      FallbackComponent={RootErrorFallback}
-      resetKeys={[router.asPath]}
-      onReset={useQueryErrorResetBoundary().reset}
-    >
-      {getLayout(<Component {...pageProps} />)}
-    </ErrorBoundary>
+    // NOTE: This suspense tag provides a convenient global fallback
+    // Make sure to add suspense lower in the tree to optimize loading experience where possible
```

Comment:
This is a nice touch, but it will break some of our current recipe code. So I'll remove it for now.
```suggestion
```